packages/server/__tests__/run_arpa_reporter_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Have you got an action item tracked anywhere to clean this up post-merge?

Reply:
Not explicitly, though we don't technically need to do anything with this; I suppose it's a slight readability improvement to remove a level of indirection.

Eventually, it might be nice to have GOST and ARPA tests use the same Mocha instance, same seeds, etc. There are some other things to reconcile there, e.g. ARPA's route tests start a server in the same process and use Supertest to interact with it but GOST's start a server in a separate process. I like ARPA's approach better (I guess I would since I added it 😛) since:
1. It doesn't require you to stop your dev server to run tests (in GOST, they compete for port 3000)
2. You could theoretically mock/spy on things since it's in the same process (can't do this in GOST)
3. Easier to use a debugger with the test + tested code if they're in the same process